spinta/formats/rdf/commands.py

```diff
@@ -0,0 +1,388 @@
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    return Cell(value.render(), about=_get_model_link(value), prefix=prefix)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, DataType, type(None))
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: DataType,
+    value: None,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    return Cell('', prefix=prefix)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, DataType, object)
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: DataType,
+    value: Any,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    if dtype.prop.name == '_id':
+        return Cell(value, about=_get_model_link(
+            dtype.prop.model,
+            pk=value,
+        ))
+    return Cell(value, prefix=prefix)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, Date, datetime.date)
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: Date,
+    value: datetime.date,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    return Cell(value.isoformat(), prefix=prefix)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, Time, datetime.time)
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: Time,
+    value: datetime.time,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    return Cell(value.isoformat(), prefix=prefix)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, DateTime, datetime.datetime)
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: DateTime,
+    value: datetime.datetime,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    return Cell(value.isoformat(), prefix=prefix)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, Number, Decimal)
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: Number,
+    value: Decimal,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    return Cell(value, prefix=prefix)
+
+
+@commands.prepare_dtype_for_response.register(Context, Rdf, Ref, (dict, str, type(None)))
+def prepare_dtype_for_response(
+    context: Context,
+    fmt: Rdf,
+    dtype: File,
+    value: NotAvailable,
+    *,
+    data: Dict[str, Any],
+    action: Action,
+    select: dict = None,
+    prefix: str = None
+):
+    super_ = commands.prepare_dtype_for_response[Context, Format, Ref, dict]
+    data = super_(context, fmt, dtype, value, data=data, action=action, select=select)
+    data['prefix'] = prefix
+    return data
```

Comment:
Ref properties should be rendered as

```xml
<prefix:name rdf:resource="{value['_id']}" />
```

If only `_id` is given (normalized form), but for denormalized form, we need to do full value traversal:

```xml
<prefix:name>
  <prefix:RefModel rdf:about="{value['_id']}">
    <prefix:prop ...>
  </prefix:RefModel>
</prefix:name>
```